modules/components/src/DataTable/utils.js

```diff
@@ -20,18 +20,33 @@ export function normalizeColumns({
     ...columnTypes,
     ...customTypes,
   };
+
   const mappedColumns = columns
-    .map(column => ({
-      ...column,
-      show: typeof column.show === 'boolean' ? column.show : true,
-      Cell: column.Cell || types[column.type],
-      hasCustomType: isNil(column.hasCustomType)
-        ? !!(customTypes || {})[column.type]
-        : column.hasCustomType,
-      ...(!column.accessor && !column.id ? { id: column.field } : {}),
-      ...(customTypeConfigs[column.type] || {}),
-    }))
+    .map(column => {
+      const customColIndex = customColumns.findIndex(
+        cc => cc.content.field === column.field,
+      );
+
+      let customCol = {};
```

Comment:
I think we can achieve this with:
```
const customCol = customColumns.find(cc => cc.content.field === column.field)?.content || {}
```
Then, I don't think there's a need for the `if` bellow, but please correct me if I'm wrong :P

Reply:
yup tested :)